lib/ansible/parsing/vault/__init__.py

```diff
@@ -1473,3 +1511,14 @@ def decrypt(cls, b_vaulttext, secret):
     u'AES': VaultAES,
     u'AES256': VaultAES256,
 }
+CIPHER_WRITE_BLACKLIST = [
+    u'AES'
+]
+CIPHER_BLACKLIST = []
+CIPHER_WHITELIST = frozenset(
+    [item for item in CIPHER_MAPPING.keys() if item not in CIPHER_BLACKLIST]
+)
+CIPHER_WRITE_WHITELIST = frozenset(
+    [item for item in CIPHER_MAPPING.keys() if item not in CIPHER_WRITE_BLACKLIST]
+)
+# Also see DEFAULT_CIPHERS at the top of this file
```

Comment:
We should leave the blacklists and whitelists alone for now.  I don't like adding unnecessary symbols (So no blacklists, for instance).  Everything should be a frozenset.  Generating the whitelists from CIPHER_MAPPING.keys() does have some appeal but considering that there's currently only two ciphers, not very much compared to creating the blacklist unneccessarily.

Those difficulties could all be hashed out in a different PR if we want.

Reply:
ok, I'll remove it for now.

Reply:
ok, reverted the change.